kratos.gid/apps/Structural/xml/ConstitutiveLaws.xml

```diff
@@ -201,4 +201,134 @@
         </inputs>
         <outputs></outputs>
     </CLaw>
+
+    <!--Damage isotropic-->
+    <CLaw n="SmallStrainIsotropicDamage3DVonMisesVonMises" pn="Small Strain Isotropic damage"  Type="3D" Behaviour="Damage" StrainSize="6" ImplementedInApplication="StructuralMechanicsApplication" help="Isotropic damage for 3d solids" Dimension="3D" LargeDeformation="False" RequiresLocalAxes="False" HybridType="False">

```

Comment:
I'd give few more details in the help.

Reply:
done

Reply:
Maybe a bit picky, but I'd add what are these VonMisesVonMises (I guess that these would be the failure surfaces or something of this sort). This nomenclature might be obvious for you but maybe it's not for the users of the GUI 😅 .

Reply:
Ok, you're right. Gonna do it